docs/pages/getting-started/5-irma-contract.rst

```diff
@@ -0,0 +1,343 @@
+.. _irma-contract:
+
+Signing a contract with IRMA
+############################
+
+This getting started manual shows how to successfully use IRMA to sign a contract.
+Contracts are used within the Nuts ecosystem to identify a user to other network participants.
+It also connects a user to the care organization that user is currently working for.
+The signed contract can be seen as a network enabled session token and should therefore be stored alongside the user session.
+The contract is required for every request that results in personal and/or medical data being retrieved.
+
+Basic requirements
+******************
+
+To use IRMA as a means for signing a contract, the following is required:
+
+- the user has the IRMA app installed on an android or iOS device with camera and an internet connection.
+- the user interacts with the XIS/ECD via a recent browser capable of running javascript.
+- the vendor has a Nuts node running.
+
+IRMA flow
+*********
+
+We use the Nuts node as IRMA server and as tool to start an IRMA session. This follow the flow as described on this `IRMA Github page <https://github.com/privacybydesign/irma-frontend-packages#supported-irma-flows>`_.
+The XIS/ECD will have to provide two endpoints for the frontend. One endpoint to start a session and one to get the session result.
+More info on these endpoints will be provided further down.
+
+
+Configuring the Nuts node
+*************************
+
+In the contract signing flow, the device running the IRMA app communicates with the Nuts node directly.
+Therefore the Nuts node needs to be accessible to the public internet.
+All APIs on the Nuts node starting with ``/public`` must be accessible over https without any security measures in place.
```

Comment:
"over HTTPS without any additional security measures that could prevent access by mobile devices."

(HTTPS is a security measure, and a vendor could add additional security measures as long as they don't hamper the functionality)

Reply:
the footnote might be a bit too difficult for a getting started. I think it's best to say that it's plain https

Reply:
Replaced sentence with your suggestion